build.sbt

```diff
@@ -15,11 +15,12 @@ lazy val core = (project in file("core")).settings(
   resolvers ++= Resolver.sonatypeOssRepos("releases"),
   libraryDependencies ++= Seq(
     "com.madgag" %% "rate-limit-status" % "0.7",
-    "com.typesafe.play" %% "play" % "2.8.19",
-    "com.squareup.okhttp3" % "okhttp" % "4.10.0",
+    "org.playframework" %% "play" % "3.0.0",
+    "com.squareup.okhttp3" % "okhttp" % "4.12.0",
     "com.lihaoyi" %% "fastparse" % "3.0.0",
     "com.madgag" %% "scala-collection-plus" % "0.11",
     "com.madgag.scala-git" %% "scala-git" % scalaGitVersion,
+    "joda-time" % "joda-time" % "2.12.5",
```

Comment:
This used to be a transitive dependency of `play-json` but has been dropped in the newer version of Play. Presumably they've switched to using the "new" java.time API, however we still need use Joda in this library and I think switching us over is outside the scope of this PR.

Reply:
Yeah, I did wonder if we could sneak in removing Joda-Time with this PR but it looks a little harder than I originally thought - have pulled that out to https://github.com/rtyley/play-git-hub/pull/9, and for the meantime, yup, let's explicitly add in the dependency.